java/client/src/main/java/glide/api/commands/ScriptingAndFunctionsClusterCommands.java

```diff
@@ -84,4 +84,74 @@ public interface ScriptingAndFunctionsClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> functionLoadReplace(String libraryCode, Route route);
+
+    /**
+     * Invokes a previously loaded function.<br>
```

Comment:
extra <br>?

Reply:
No, it is intentionally here to put phrase about routing on the next line